audio/out/buffer.c

```diff
@@ -35,6 +35,12 @@
 #include "osdep/timer.h"
 #include "osdep/threads.h"
 
+struct queued_aframe {
+    double pts;
+    double speed;
+    double start_time;
```

Comment:
Does it makes sens to be double? Looks like it is converted from NS most of the time anyway

Reply:
Core uses double for timestamps. I guess your opinion is to keep using int64_t in audio part and do all conversions in core. I agree with this idea.